Controls/Grid/Grid.UWP/Model/TableModel/DisplayData/CellsController.SlotLength.cs

```diff
@@ -122,14 +126,14 @@ internal virtual double GetSlotHeight(int cellSlot)
         /// <returns>Returns true only if Slot Height was Updated (e.g. Smaller then the new Height).</returns>
         internal bool UpdateSlotHeight(int cellSlot, double cellHeight)
         {
-            if (this.table.RowHeightIsNaN && !this.RowPool.IsItemCollapsed(cellSlot))
+            if (this.table.RowHeightIsNaN && !this.RowPool.IsItemCollapsed(cellSlot) || this.table.HasExpandedRowDetails(cellSlot))
             {
                 var renderInfo = this.RowPool.RenderInfo;
                 var currentHeight = renderInfo.ValueForIndex(cellSlot, false);
 
-                bool isLessThan = GridModel.DoubleArithmetics.IsLessThan(currentHeight, cellHeight);
+                bool isLessThan = true;
```

Comment:
This is not changed anywhere. We could simply return true in the if statement after it.